lib/project-helpers.ts

```diff
@@ -1,6 +1,50 @@
-export interface CreateProjectValues {
-  name: string;
-  goal: string;
-  owner: string;
-  champion: string;
+import { useEffect } from "react";
+import { useState } from "react";
+import { Project } from "../db/models/project";
+
+export class ProjectValues {
+  public id?: number;
+  public name: string = "";
+  public goal: string = "";
+  public owner: string = "";
+  public champion: string = "";
+  public adminEmails: string[] = [];
+
+  public constructor(init?: Partial<ProjectValues>) {
```

Comment:
What's the reason for needing this constructor?

Reply:
I sometimes create a new `ProjectValues` with only some of the values. It's just a handy mechanism.

Reply:
You can do that with an interface type as well.

Reply:
You don't need a constructor for that. Generally, it is preferable to use an interface here and then define the variable as follows:

```
const project: Partial<ProjectValues> = {
    ...
};
```